router/router.go

```diff
@@ -1523,8 +1470,13 @@ func (rt *HandleT) commitStatusList(responseList *[]jobResponseT) {
 			userID := resp.userID
 			worker := resp.worker
 			if status != jobsdb.Failed.State {
+				if worker == nil {
+					// worker is nil when we're trying to find a worker and there is no destination configuration
+					rt.logger.Debugf("[%v Router] :: No worker was found for job %d", rt.destName, resp.status.JobID)
+					continue
+				}
```

Comment:
Why is this needed? We shouldn't call `commitStatusList` if no worker is found

Reply:
I forgot to remove it (old refactoring), thanks for catching it!